service-templates/apigw-lambda-svc/v1/pipeline_infrastructure/pipeliine.tf

```diff
@@ -0,0 +1,685 @@
+################################################################################
+# S3 Bucket - Function
+################################################################################
+
+locals {
+  environment_account_ids = split(",", var.environment_account_ids)
```

Comment:
good example of a local variable usage:
1. Its closed to where its used so you don't have to go far to understand whats what
2. Its handling some transformation/processing before its used since we cannot pass the value as we would have liked (here we split a comma separated string to get an array of strings instead of simply passing in an array of strings since CloudFormation doesn't allow - I *assume)